core/poi/lib/plugins/config-css.js

```diff
@@ -64,6 +64,10 @@ exports.apply = api => {
             rule
               .use('extract-css-loader')
               .loader(require('mini-css-extract-plugin').loader)
+              .options({
+                hmr: !api.isProd,
```

Comment:
```suggestion
                hmr: api.mode === 'development',
```

Only need it in `development` mode, FYI there's also `test` mode